global/classic-zaza/tox.ini

```diff
@@ -47,15 +47,17 @@ commands =
     {toxinidir}/rename.sh
     charmcraft clean
 
+[testenv:py38]
+basepython = python3.8
+deps = -r{toxinidir}/merged-requirements-py38.txt
+
 [testenv:py310]
 basepython = python3.10
-deps = -r{toxinidir}/requirements.txt
-       -r{toxinidir}/test-requirements.txt
+deps = -r{toxinidir}/merged-requirements-py310.txt
 
 [testenv:py3]
 basepython = python3
-deps = -r{toxinidir}/requirements.txt
```

Comment:
Out of curiosity why do the default py3 targets use py38 instead of py310?

Reply:
It's a good question, and one I pondered for a bit.  Essentially, I went for the lowest common denominator; the py3 target isn't used in the CI system(s) (they use the py38 and py310 targets directly), so it is essentially a fallback target.  I did think about removing it, but considered that even more painful!